### PR TITLE
README docs for python and nodejs

### PR DESCRIPTION
**Pull request by maciekrb**
Adds usage documentation for python and nodejs

### CODE REVIEW

File: nodejs/README.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+```js
+# Configure the driver (base port) socket
+var configSocket = zmq.socket('push')
+configSocket.connect('tcp://' + creator_ip + ':' + creator_humidity_base_port)
+
+// Now prepare valid configuration and send it.
+var config = matrix_io.malos.v1.driver.DriverConfig.create({
+  delayBetweenUpdates: 2.0,  // 2 seconds between updates
+  timeoutAfterLastPing: 6.0, // Stop sending updates 6 seconds after pings.
+  humidity: matrix_io.malos.v1.sense.HumidityParams.create({
```
Comment (eighteyes) [inline nodejs/README.md]: this adds complexity and isn't part of every sensor

File: .gitignore [hunk 2]
```diff
@@ -1 +1,7 @@
 build/
+nodejs/index.js
+nodejs/node_modules/
```
Comment (maciekrb) [inline .gitignore]: It's a precaution :P so they don't get added to the repo. It has happened before.

File: .travis.yml [hunk 3]
```diff
@@ -3,24 +3,11 @@ dist: jessie
 language: cpp
 cache: ccache
 
-env:
-  matrix:
-    - GRPC_RELEASE_TAG=v1.1.4
-    - GRPC_RELEASE_TAG=v1.2.0
-
 before_install:
+  - echo "deb http://unstable-packages.matrix.one/ stable main" | sudo tee /etc/apt/sources.list.d/matrixio.list
```
Comment (maciekrb) [inline .travis.yml]: Yeah,  so this is how the next iteration is looking like : 
```
deb http://packages.matrixlabs.ai/ubuntu trusty main
deb http://packages.matrixlabs.ai/debian jessie main
```

File: nodejs/README.md (hunk 1 above)
Comment (maciekrb) [inline nodejs/README.md]: Removing from humidity for simplicity, documenting better for pressure.

File: .travis.yml [hunk 4]
```diff
@@ -3,24 +3,11 @@ dist: jessie
 language: cpp
 cache: ccache
```
Comment (maciekrb) [inline .travis.yml]: Added a ref to official travis docs regarding caching

**Pull request merged**